less/angular/pages/private-eye.less

```diff
@@ -1,44 +1,4 @@
-.private-eye-footer {
-  text-align: center;
+.cta-footer {
```

Comment:
Possible to move this style into an existing file? Maybe line 55 of less/angular/components/footer.less?

If not, it's not a huge deal.